aeron-cluster/src/main/java/io/aeron/cluster/ConsensusModule.java

```diff
@@ -1306,6 +1308,9 @@ public void conclude()
                 egressPublisher = new EgressPublisher();
             }
 
+            final ChannelUri channelUri = ChannelUri.parse(logChannel());
+            logChannelIsMultiDestination = channelUri.isUdp() && null == channelUri.get(ENDPOINT_PARAM_NAME);
```

Comment:
Boolean naming convention starts with `is` or `has`, therefore `isLogChannelMultiDestination`.